endpoints.php

```diff
@@ -1114,7 +1114,11 @@ function updateTapestryNodeCoordinates($request)
             throw new TapestryError('INVALID_NODE_META_ID');
         }
         if (!TapestryHelpers::userIsAllowed('EDIT', $nodeMetaId, $postId)) {
-            throw new TapestryError('EDIT_NODE_PERMISSION_DENIED');
+            if (TapestryHelpers::userIsAllowed('MOVE', $nodeMetaId, $postId)) {
+               // Authors are allowed to move their nodes but not edit after submission
+            } else {
+                throw new TapestryError('EDIT_NODE_PERMISSION_DENIED');
+            }
```

Comment:
Reverse conditional to avoid empty statements
```suggestion
            if (!TapestryHelpers::userIsAllowed('MOVE', $nodeMetaId, $postId)) {
                throw new TapestryError('EDIT_NODE_PERMISSION_DENIED');
            }
```
I would also suggest changing `EDIT_NODE_PERMISSION_DENIED` to a newly created `MOVE_NODE_PERMISSION_DENIED` error to be more accurate.